groups/stack/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,40 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-development-terraform-state-london"
+environment = "cidev"
+deploy_to = "development"
+state_prefix = "env:/development"
+aws_profile = "development-eu-west-2"
+
+cookie_domain = "cidev.aws.chdev.org"
+cookie_name = "__SID"
+
+# Certificate for https access through ALB
+ssl_certificate_id = "arn:aws:acm:eu-west-2:169942020521:certificate/8d7db053-7416-4e56-946b-762d0a34c899"
+zone_id = "Z2KSI4Z5ZN9NT0"
```

Comment:
These could be replaced with data sources, removing the need to hard-code these values.

Reply:
do you mean something like:
`zone_id = data.aws_route53_zone.zone[0].zone_id`
I got this by searching for zone_id in CH repos.  But I'm not sure what I need.